docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html

```diff
@@ -0,0 +1,62 @@
+<nav class="bg-white p4 mn2">
+  <div class="row">
+    <div class="col-fit h8 row-end">
+      <div class="h6 row-center-x">
+        {% render '@icon-square--home-active-large'%}
```

Comment:
I'd suggest removing the sizing here `large` in favor of letting the block the icon is in define the size.